docs/setup.md

```diff
@@ -17,30 +17,48 @@
 
 The configuration settings of the FortiAI Configuration widget include the following steps to setup FortiAI configuration:
 
-- Click the **Configure** button to bring up the following wizard screen
+1. Click the **Configure** button to bring up the following wizard screen
 
-    ![FortiAI Configration](./res/fortiai_start.png)
+    ![FortiAI Configuration](./res/fortiai_start.png)
 
-- Click **Let's Get Started** to proceed
+2. Click **Let's Get Started** to proceed.
 
-- On **Configuration** page, select the following fields
-  - LLM Integration &mdash; **OpenAI**.
-  - Conversation Model &mdash; **gpt-3.5-turbo, gpt-3.5-turbo-0301, gpt-4, gpt-4-1106-preview**.
-  - Playbook Generation Model &mdash; **gpt-3.5-turbo, gpt-3.5-turbo-0301, gpt-4, gpt-4-1106-preview**.
-  - Enable Per-user LLM configuration.
+3. On the **Configuration** page, select a value for the following fields:
 
-- Click **Next** on lower-right corner
+    ![](./res/fortiai_configuration.png)
+
+    - **LLM Integration** &mdash; *OpenAI* (default).
+    - **Conversation Model**: Select one from the following options:
+
+        - `gpt-4o-2024-05-13`
+        - `gpt-4o`
+        - `gpt-4-1106-preview`
+        - `gpt-4`
 
-    ![Configuration](./res/fortiai_configuration.png)
+    - **Playbook Generation Model**: Select one from the following options:
 
-- On the **Connect LLM** page, configure your LLM Integration.
+        - `gpt-4o-2024-05-13`
+        - `gpt-4o`
+        - `gpt-4-1106-preview`
+        - `gpt-4`
```

Comment:
Please add note here saying these list keeps changing and can be modified by following steps;
@cs-ankit-gupta  can provide you steps to modify this list and this section will also be part of SP document.